Dev parameters hover over (#268)
neo4j/docs-status-codes: modules/ROOT/pages/errors/gql-errors/index.adoc

=== xref:errors/gql-errors/22N12.adoc[22N12]

Status description:: error: data exception - invalid date, time, or datetime format. Invalid argument: cannot process `{ <<input>> }`.

=== xref:errors/gql-errors/22N13.adoc[22N13]

Status description:: error: data exception - invalid time zone. Specified time zones must include a date component.

=== xref:errors/gql-errors/22N14.adoc[22N14]

Status description:: error: data exception - invalid temporal value combination. Cannot select both `{ <<temporal>>1 }` and `{ $temporal2 }`.

=== xref:errors/gql-errors/22N15.adoc[22N15]

Status description:: error: data exception - invalid temporal component. Cannot read the specified `{ <<component>> }` component from `{ <<temporal>> }`.

=== xref:errors/gql-errors/22N18.adoc[22N18]

Status description:: error: data exception - incomplete spatial value. A `{ <<crs>> }` POINT must contain `{ <<mapKeyList>> }`.

=== xref:errors/gql-errors/22N19.adoc[22N19]

Status description:: error: data exception - invalid spatial value. A `POINT` must contain either 'x' and 'y', or 'latitude' and 'longitude'.

=== xref:errors/gql-errors/22N20.adoc[22N20]

> Status description:: error: data exception - invalid spatial value dimensions. Cannot create POINT with `{ <<dim>>1 }` coordinate reference system (CRS) and `{ <<value>> }` coordinates. Use the equivalent `{ $dim2 }` coordinate reference system instead.

Reviewer: ```suggestion
Status description:: error: data exception - invalid spatial value dimensions. Cannot create POINT with `{ <<dim>>1 }`D coordinate reference system (CRS) and `{ <<value>> }` coordinates. Use the equivalent `{ $dim2 }`D coordinate reference system instead.
```

Looks like the D:s have fallen out, the parameter is e.g. 2 or 3 so it becomes 2D or 3D

> Status description:: error: data exception - invalid spatial value dimensions. Cannot create POINT with `{ <<dim>>1 }`D coordinate reference system (CRS) and `{ <<value>> }` coordinates. Use the equivalent `{ $dim2 }`D coordinate reference system instead.

Author: ```suggestion
Status description:: error: data exception - invalid spatial value dimensions. Cannot create POINT with `{ <<dim>>1D }` coordinate reference system (CRS) and `{ <<value>> }` coordinates. Use the equivalent `{ $dim2D }` coordinate reference system instead.
```

=== xref:errors/gql-errors/22N74.adoc[22N74]

Status description:: error: data exception - index conflicts with existing constraint. An index that belongs to the constraint `{ <<constr>> }` contains a conflicting index.

=== xref:errors/gql-errors/22N75.adoc[22N75]

Status description:: error: data exception - constraint contains duplicated tokens. The constraint specified by `{ <<constrDescrOrName>> }` includes a label, relationship type, or property key with name `{ <<token>> }` more than once.

=== xref:errors/gql-errors/22N76.adoc[22N76]

Status description:: error: data exception - index contains duplicated tokens. The index specified by `{ <<idxDescrOrName>> }` includes a label, relationship type, or property key with name `{ <<token>> }` more than once.

=== xref:errors/gql-errors/22N77.adoc[22N77]

Status description:: error: data exception - property presence verification failed. `{ <<entityType>> }` (`{ <<entityId>> }`) with `{ <<tokenType>> }` `{ <<token>> }` must have the following properties: `{ <<propKeyList>> }`.

=== xref:errors/gql-errors/22N78.adoc[22N78]

> Status description:: error: data exception - property type verification failed. `{ <<entityType>> }` (`{ <<entityId>> }`) with `{ <<tokenType>> }` `{ <<token>> }` must have the property `$propKey` with value type `{ <<valueType>> }`.

Reviewer: ```suggestion
Status description:: error: data exception - property type verification failed. `{ <<entityType>> }` (`{ <<entityId>> }`) with `{ <<tokenType>> }` `{ <<token>> }` must have the property `{ <<propKey>> }` with value type `{ <<valueType>> }`.
```

=== xref:errors/gql-errors/22NB0.adoc[22NB0]

Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ <<pred>> }` always evaluates to 'NULL'. Use `WHERE` syntax in combination with `IS NULL` instead.

=== xref:errors/gql-errors/22NB1.adoc[22NB1]

Status description:: error: data exception - type mismatch. Type mismatch: expected to be `{ <<valueTypeList>> }` but was `{ <<input>> }`.

=== xref:errors/gql-errors/22NB2.adoc[22NB2]

Status description:: error: data exception - incompatible graph type dependence. Graph type `{ <<graphTypeDependence>>1 }` constraint `{ <<constrDescrOrName>>1 }` is incompatible with graph type `{ $graphTypeDependence2 }` constraint `{ $constrDescrOrName2 }` because they have different graph type dependence.

=== xref:errors/gql-errors/22NB3.adoc[22NB3]

Status description:: error: data exception - token presence verification failed. `{ <<entityType>> }` (`{ <<entityId>> }`) with `{ <<tokenType>>1 }` `{ <<token>>1 }` must have the `{ $tokenType2 }` `{ $token2 }`.

=== xref:errors/gql-errors/22NB4.adoc[22NB4]

> Status description:: error: data exception - endpoint label presence verification failed. Relationship (`{ <<entityId>>1 }`) with type `{ <<relType>> }` requires its `{ $endpointType }` node (`{ $entityId2 }`) to have the label `{ <<label>> }`.

Reviewer: ```suggestion
Status description:: error: data exception - endpoint label presence verification failed. Relationship (`{ <<entityId>>1 }`) with type `{ <<relType>> }` requires its `{ <<endpointType>> }` node (`{ $entityId2 }`) to have the label `{ <<label>> }`.
```

=== xref:errors/gql-errors/42I41.adoc[42I41]

Status description:: error: syntax error or access rule violation - invalid use of variable length relationship. Variable length relationships cannot be used in `{ <<value>> }`.

=== xref:errors/gql-errors/42I45.adoc[42I45]

Status description:: error: syntax error or access rule violation - invalid use of multiple path patterns. Multiple path patterns cannot be used in the same clause in combination with a selective path selector. `{ <<action>> }`

=== xref:errors/gql-errors/42I47.adoc[42I47]

Status description:: error: syntax error or access rule violation - parser error. Parser Error: `{ <<msg>> }`.

=== xref:errors/gql-errors/42I48.adoc[42I48]

Status description:: error: syntax error or access rule violation - invalid use of a subquery in MERGE. Subqueries are not allowed in a MERGE clause.

=== xref:errors/gql-errors/42I50.adoc[42I50]

> Status description:: error: syntax error or access rule violation - token name too long. Invalid input `{ <<input>> }`..`. A `{ <<tokenType>> }` name cannot be longer than `{ $maxTokenLength }`.

Author: Ah, yes. Good catch!

> Status description:: error: syntax error or access rule violation - token name too long. Invalid input `{ <<input>> }`..`. A `{ <<tokenType>> }` name cannot be longer than `{ $maxTokenLength }`.

Author: ```suggestion
Status description:: error: syntax error or access rule violation - token name too long. Invalid input `{ <<input>> }..`. A `{ <<tokenType>> }` name cannot be longer than `{ <<value>> }`.
```
